driverapi/driverapi.go

```diff
@@ -104,6 +104,10 @@ type JoinInfo interface {
 	// SetGatewayIPv6 sets the default IPv6 gateway when a container joins the endpoint.
 	SetGatewayIPv6(net.IP) error
 
+	// SetStaticRoutes sets the routes that should be added to the sandbox.  It
+	// may be used in addtion to or instead of a default gateway (as above).
+	SetStaticRoutes(routes []types.StaticRoute) error
```

Comment:
One common guideline that we have tried to maintain is to not expose structures in the interface. So instead of `SetStaticRoutes` accepting a type can we change it to `AddStaticRoute(routeType, Prefix, nh, interfaceID)` kind of an api which the driver can call multiple times to add as many routes? This will then work very similar to `AddInterface` api.